subcell_pipeline/analysis/compression_metrics/polymer_trace.py

```diff
@@ -226,9 +223,182 @@ def get_bending_energy_from_trace(
     return energy.item()
 
 
+def get_2d_polymer_trace(
+    polymer_trace: np.ndarray,
+    compression_axis: int = 0,
+) -> np.ndarray:
+    """
+    Get the 2D projection of the polymer trace.
+
+    Parameters
+    ----------
+    polymer_trace
+        The polymer trace as an Nx3 numpy array.
+
+    compression_axis
+        The axis along which the polymer trace is compressed.
+
+    Returns
+    -------
+    return1: type
+        return variable description
+    """
+    if polymer_trace.shape[1] == 2:
+        return polymer_trace
+
+    return polymer_trace[
+        :, [ax for ax in range(polymer_trace.shape[1]) if ax != compression_axis]
+    ]
+
+
+def get_normalized_tangent_vectors(
+    polymer_trace: np.ndarray,
+) -> np.ndarray:
+    """
+    Calculate the normalized tangent vectors for a polymer trace.
+
+    Parameters
+    ----------
+    polymer_trace
+        The polymer trace as an Nx3 numpy array.
+
+    Returns
+    -------
+    return1: type
+        return variable description
+    """
+    tangents = np.diff(polymer_trace, axis=0)
+
+    tangents /= np.linalg.norm(tangents, axis=1)[:, np.newaxis]
+
+    return tangents
+
+
+def get_twist_angle(
+    polymer_trace: np.ndarray,
+    **options: Dict[str, Any],
+) -> float:
+    """
+    Calculate the twist angle of the polymer trace.
+
+    Parameters
+    ----------
+    polymer_trace
+        array containing the x,y,z positions of the polymer trace
```

Comment:
nit: Make sure parameter descriptions start with capital letter and ends with period for consistency